### ISSUE TITLE
Accessibility: Unable to open search bar without using mouse

### ISSUE THREAD

**Issue by AlexGStapleton** [siteorigin/siteorigin-north#170]
This is a major issue for those physically unable to use a mouse.

**Comment by gregpriday** [comment]
Not sure how we'd go about this, but this is a great usability enhancement.

**Comment by adiraomj** [comment]
@AlexGStapleton we will try to address this issue in #123

**Comment by Misplon** [comment]
@adiraomj Perhaps we could take another look.

**Comment by Misplon** [comment]
@adiraomj Do you have any ideas for how the header search might be opened with the keyboard? Perhaps in tab order? We can also implement this in Unwind.

**Comment by Misplon** [comment]
Resolved via: https://github.com/siteorigin/siteorigin-north/commit/5e6ed52c29ea5d4f24007c75ada3c878f50f5515.
